pkgs/data/fonts/et-book/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub }:
+
+fetchFromGitHub rec {
+  rev = "1.0";
```

Comment:
Since upstream is not versioned, this should be `2018-09-10`

Reply:
using the rev string instead, would that be okay?